pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
This seems a bit complex for my tastes, but I'm simply not qualified to comment on go packaging, sorry.